src/dps.c

```diff
@@ -1187,14 +1252,25 @@ DPS_Node* DPS_CreateNode(const char* separators, DPS_KeyStore* keyStore, const D
     /*
      * Sanity check
      */
-    if (keyId && (!keyStore || !keyStore->keyHandler)) {
+    if (id && (!keyStore || !keyStore->keyHandler)) {
```

Comment:
Is a zero length id permitted - should a minimum id length be enforced - it used to be 128 bits.

Reply:
I would not expect a 0-length id, although it's ultimately up to the key store implementation what it's going to do with it: is it going to return a valid key when passed a zero-length id?

I'm reluctant to put a minimum as that will limit applications on what they can use as CN values in their certificates.